cmd/format-release-notes/variables.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+var iconsMap = map[string]string{
```

Comment:
JFYI: Speaking of those icons, we finally got rid of them in v3. https://github.com/percona/pmm/blob/v3/documentation/variables.yml